### PR TITLE
Milestone 2

### PR DESCRIPTION
**Pull request by julian3493**
### Initialize redux

- Implement App component wrapping BooksList and BooksForm components
- Implement BookList component
- Implement BooksForm component
- Implement Book component
- Define books reducer
- create create_book and remove_book actions

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,11 +1,24 @@
...
+];
+
+const store = createStore(index, { books: initialState });
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Provider store={store}>
+      <App />
+    </Provider>
```
Comment (eliasecasta) [inline src/index.js]: * [ ] It wraps `App` in `Provider` and assigns the whole thing into a variable (Currently, you are only missing to assign this to a variable and use it accordingly)

**Review by eliasecasta — CHANGES_REQUESTED** [review]
# Status:  Changes Required ♻️
_Check the comments under the review_
<hr>

Hello @julian3493, I am Elias, your reviewer for this PR

Well done so far with your project!

I will mention to you a small summary of things that need to be changed in order to keep moving:

- [ ] The only thing that you missed was to assign your nested components to one variable and use it to comply with the project requirements, please do so in order to move on with this milestone.

Please correct those issues and send another review request, good luck, and well done so far! 

Code review by Elias Castañeda

_(Feel free to leave any questions or comments in the PR thread if something is not 100% clear)_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/index.js (hunk 1 above)
Comment (julian3493) [reply src/index.js]: I can't understand exactly what I must assign to a variable, I review other student's repos for this project and I can't see any difference or hint related to that.

Comment (eliasecasta) [reply src/index.js]: Hello Julian, this is what Microverse wants you to implement: 
image

**Review by eliasecasta — APPROVED** [review]
## Status:  Approved 🟢

Hi, @julian3493, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**